### PR TITLE
fix: Human-Template-include-items, Minor Compendium Errors, Multiple rqids with je..track

### LINKED ISSUE #481: Human Template include items
### _Why:_ Is your feature request related to a problem?

The human template does not include items properly.  Use the same item include functionality to include skills, runes, hit locations, etc.

Other related considerations:

1. Remove the "species-templates" compendium and move to "bestiary"?
2. Should there be a human template for each Homeland where we override the base chance of skills (RQG p 60-63) and add the "Homeland Base Passions" (RQG p 27), and anything else that changes between homelands?
3. Does it make sense to have "Player Character" templates that have some more things on them than NPC templates?

### _What:_ Describe the solution you'd like

See above

### _Options:_ Describe alternatives you've considered

n/a

### Additional context

n/a

### PR DESCRIPTION
**Pull request by Moonpile**
Fixes #481

- Used the Sartar text from the wiki in the Human (Sartarite)

### CODE REVIEW

**Comment by Moonpile** [comment]
I created a vanilla "Human (Base)" and moved it to the bestiary.  This gets rid of the species-templates pack and breaks the Adventurer Wizard, but that is already well-broken.  I also created a "Human (Sartarite)" that has the cultural skills, rune, and passions pre-applied.  I didn't do the other homelands on the thinking that they will get included in the RQG module.

The "Customs ()" skills were using homelands.  I changed it to use cultures and agree with the skills as named in the "Homeland Cultural Skill Bonuses" from RQG p 60-63.  

Fixed the Kick skill name.

Removed the ui.notificication.errors in applyFnToItemFromHandlebarsArgs

**Comment by Moonpile** [comment]
I accidentally fixed #482 on the same branch so I added it to this PR.

File: src/i18n/en/openSystem.json [hunk 1]
```diff
@@ -2527,6 +2539,13 @@
       "bite": {
         "name": "Bite"
       },
+      "atlatl": {
```
Comment (wake42) [inline src/i18n/en/openSystem.json:2542]: Should be plural
```suggestion
      "atlatls": {
```

**Review by wake42 — CHANGES_REQUESTED** [review]
demo-actors & demo-items compendiums are still defined in system.json resulting in empty compendiums so they need to be removed.

**Comment by Moonpile** [comment]
> demo-actors & demo-items compendiums are still defined in system.json resulting in empty compendiums so they need to be removed.

If we have a pack defined in system.json and there's not a corresponding directory in pack-templates, should we be getting a warning during build?  If so I think that can be a chore issue for another version.

File: src/i18n/en/openSystem.json (hunk 1 above)
Comment (Moonpile) [reply src/i18n/en/openSystem.json]: Fixed

**Review by wake42 — APPROVED** [review]

**Pull request merged**